src/deviceconfig.cpp

```diff
@@ -96,6 +96,13 @@ DeviceConfig::DeviceConfig()
         "The hostname or IP address of the NTP server to be used for time synchronization.",
         SettingSpec::SettingType::String
     );
+    settingSpecs.emplace_back(
+        NAME_OF(rememberCurrentEffect),
+        "Remember current effect",
+        "A boolean that indicates if the current effect index should be saved after an effect transition, so the device resumes "
+        "from the same effect when restarted. Enabling this will lead to more wear on the flash chip of your device.",
+        SettingSpec::SettingType::String

```

Comment:
I think this should be Boolean.  I have generated a PR to update.  I forgot I could comment on closed PRs.